lib/bob_ross/assets/bob-ross.js

```diff
@@ -0,0 +1,130 @@
+const hmacs = {}
+const configuration = {}
+
+export function setHmac(transformation, hmac){
+  hmacs[transformation] = hmac;
+}
```

Comment:
This setter method excludes the encoding of the transformation, which is a bit annoying. Is there a way to somehow include that in this module?

This is from previous usage:
```javascript
let hmacs = {<%= [
   {resize: '64x64*',  watermark: {},   optimize: true},
   {resize: '400x400*',  watermark: {},   optimize: true},
   {resize: '400x400',  watermark: false,   optimize: true},
   {resize: '400x200',  watermark: false,   optimize: true},
   {transparent: true, resize: '150x70#',  watermark: false,   optimize: true},
   {transparent: true, padding: '20,0,20,0', resize: '150x70#',  watermark: false,  optimize: true},
   {resize: '120x90*',  watermark: false,   optimize: true}
].map { |t|
  if t[:resize]
    [1,2,3].map { |multiplier|
      size = t[:resize].gsub(/\d+/) { |d| d.to_i * multiplier }
      key = BobRoss.encode_transformations(t.merge(resize: size))
      "'#{key}': '#{BobRoss.calculate_hmac(key)}'"
    }.join(',')
  else
    key = BobRoss.encode_transformations(t)
    "'#{key}': '#{BobRoss.calculate_hmac(key)}'"
  end
}.join(', ') %>}
```

We miss out on `BobRoss.encode_transformations` and the automated inclusion of the muliplier `[1,2,3]`.